tensorboard/webapp/experiments/store/experiments_selectors_test.ts

```diff
@@ -47,4 +48,37 @@ describe('experiments selectors', () => {
       );
     });
   });
+
+  describe('#getExperimentNames', () => {
+    let state: State;
+
+    beforeEach(() => {
+      const foo = buildExperiment({id: 'foo', name: 'foo name'});
+      const bar = buildExperiment({id: 'bar', name: 'bar name'});
+
+      state = buildStateFromExperimentsState({
+        data: {
+          experimentMap: {foo, bar},
+        },
+      });
+    });
+
+    it('translates experiment ids to experiment names', () => {
+      expect(
+        selectors.getExperimentNames(['foo', 'bar', 'baz'])(state)
+      ).toEqual({
+        foo: 'foo name',
+        bar: 'bar name',
+      });
+    });
+
+    it('does not include experiments which are not found', () => {
```

Comment:
This is already tested in the last test with the 'baz' id. Maybe this should test that an empty object is returned when none of the experiments are found?

Reply:
That's a good point. I already tested that empty object is returned when nothing is found in the test on line 80

Reply:
Line 80 tests an empty object is returned with nothing is PROVIDED. Which I would say is significantly different than if nothing is found.